1 table.sql

```diff
@@ -0,0 +1,23 @@
+use InstrumentDB 
+go
+drop table if exists Instrument
+go 
+create table dbo.Instrument(
+InstrumentID int not null identity primary key ,
+CustomerFirstName varchar (50) not null constraint ck_Customer_First_Name_cannot_be_blank check (CustomerFirstName<>''),
+CustomerLastName varchar (50) not null constraint ck_Customer_Last_Name_cannot_be_blank check (CustomerLastName <>''),
+InstrumentName varchar(50) not null constraint ck_Instrument_Name_cannot_be_blank check(InstrumentName<>''),
+Category varchar(50) not null constraint ck_Instrument_Category_Cannot_be_blank check (Category<>''),
+InstrumentCode as concat (substring(CustomerFirstName,1,3),substring(Category,1,3)) persisted,
+MonthlyRentalFee decimal (5,2) not null constraint ck_Monthly_Rental_Fee_must_be_greater_then_zero check (MonthlyRentalFee >0),
+DateRented datetime  not null,
+Datereturned datetime,
+Profit as case when Datereturned is null then null
```

Comment:
This works if we say there is no partial month rentals. I guess that will be the criteria.